gatsby-node.js

```diff
@@ -1291,6 +1291,12 @@ const createRedirects = ({ actions }) => {
     isPermanent: true,
   });
 
+  createRedirect({
```

Comment:
hmm this redirect doesn't seem to be working :confused: See this for example: https://mdr-ci.staging.k6.io/docs/refs/pull/686/merge/using-k6/options/#console-output (with current docs: https://k6.io/docs/using-k6/options/#console-output)

Reply:
As far as we can tell, the redirects only work in production and on staging.k6.io.

Mihail investigated with a different redirect:

> https://mdr-ci.staging.k6.io/docs/refs/pull/686/merge/using-k6/options/#dns this doesn't work
https://k6.io/docs/cloud/project-and-team-management/team-members/#managing-members - works in production (it replaces team-members with `members
https://mdr-ci.staging.k6.io/docs/refs/pull/686/mergecloud/project-and-team-management/team-members/#managing-members also doesn't work